### PR TITLE
Sticky Notes Automation

### PR DESCRIPTION
**Pull request by karnandohuang**
please review, thanks!

### CODE REVIEW

File: src/test/resources/features/core/StickyNotes/EditStickyNotes.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      | password | administratorfunctionapp |
+    And user should be in homepage
+
+  @Positive @GoToEditStickyNotesAsAdmin
+  Scenario: user go to edit sticky note as admin
+    Given user should be logged in
+    And user should see menu bar
+    And user should be on sticky notes detail page
+    When user click on edit button
+    Then user should be on edit sticky notes page
```
Comment (oliverrsebastian) [inline src/test/resources/features/core/StickyNotes/EditStickyNotes.feature]: can you try to edit the sticky notes and see if the result in the sticky notes detail is changed?

File: src/main/java/com/future/function/qa/pages/core/StickyNotes/EditStickyNotesPage.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.future.function.qa.pages.core.StickyNotes;
+
+import com.future.function.qa.model.UrlConstant;
+import com.future.function.qa.pages.base.MenuPage;
+import net.thucydides.core.annotations.DefaultUrl;
+
+@DefaultUrl(UrlConstant.EDIT_STICKY_NOTES_URL)
+public class EditStickyNotesPage extends MenuPage {
+}
```
Comment (oliverrsebastian) [inline src/main/java/com/future/function/qa/pages/core/StickyNotes/EditStickyNotesPage.java:27]: please add more elements in this page if decided to add edit action automation on sticky notes

File: src/test/resources/features/core/StickyNotes/EditStickyNotes.feature (hunk 1 above)
Comment (karnandohuang) [reply src/test/resources/features/core/StickyNotes/EditStickyNotes.feature]: done

File: src/main/java/com/future/function/qa/pages/core/StickyNotes/EditStickyNotesPage.java (hunk 2 above)
Comment (karnandohuang) [reply src/main/java/com/future/function/qa/pages/core/StickyNotes/EditStickyNotesPage.java:27]: done

**Review by oliverrsebastian — DISMISSED** [review]

**Review by oliverrsebastian — APPROVED** [review]

**Pull request merged**